### DISCUSSION TITLE
Execution failed for task ':stripe_android:compileDebugJavaWithJavac'.

### DISCUSSION THREAD

**Discussion by ghost** [flutter-stripe/flutter_stripe#1602] [Q&A]
Execution failed for task ':stripe_android:compileDebugJavaWithJavac'.

error: invalid source release: 17

I get this error if I am using
flutter_stripe: ^10.0.0

IMAGE :( https://prnt.sc/KUg5LD0gorvq )

Please help me Thanks.

**Comment by hamzaali163** [comment]
same issue

**Comment by geraldlee2001** [comment]
same issue with this, just suddenly become this
